.github/workflows/acceptance_run.yml

```diff
@@ -4,18 +4,47 @@ on:
     - cron: "0 0 * * *"  # At 00:00 every day
   workflow_dispatch:
     inputs:
-      test_label_regexp:
-        required: false
+      build_preset:
+        type: choice
+        default: "relwithdebinfo"
+        description: "Build preset"
+        options:
+          - debug
+          - release
+          - relwithdebinfo
+          - release-asan
+          - release-tsan
+          - release-msan
+      runner_label:
         type: string
+        default: "auto-provisioned"
+        description: "runner label"
+      put_build_results_to_cache:
+        type: boolean
+        default: true
 
 jobs:
-  ya_x86_64:
-    name: Build/test x86_64 using YA
-    uses: ./.github/workflows/build_and_test_ya_provisioned.yml
-    secrets: inherit
-    with:
-      runner_label: auto-provisioned
-      build_preset: relwithdebinfo
-      build_target: ydb/tests/acceptance
-      run_build: true
-      run_tests: true
+  main:
+    name: Build and test ${{ inputs.build_preset }}    
+    runs-on: [ self-hosted, "${{ inputs.runner_label }}"]
```

Comment:
Apart from `auto-provisioned`, you should use the `build-preset-<preset_name>` label here. You can look at how this label is generated [here](https://github.com/ydb-platform/ydb/blob/main/.github/workflows/build_and_test_ya_provisioned.yml#L104). It is needed because different presets require different VM configurations..

Reply:
Done